src/test/java/org/opensearch/security/dlic/rest/api/MultiTenancyConfigApiTest.java

```diff
@@ -54,9 +57,43 @@ private void verifyTenantUpdate(final Header... header) throws Exception {
             setPrivateTenantAsDefaultResponse.getStatusCode(),
             equalTo(HttpStatus.SC_OK)
         );
+        assertThat(getDashboardsinfoResponse.findArrayInJson("sign_in_options"), hasItem(DashboardSignInOption.BASIC.toString()));
+        assertThat(getDashboardsinfoResponse.findArrayInJson("sign_in_options"), not(hasItem(DashboardSignInOption.SAML.toString())));
+        assertThat(getDashboardsinfoResponse.findArrayInJson("sign_in_options"), not(hasItem(DashboardSignInOption.OPENID.toString())));
+
+        final HttpResponse updateDashboardSignInOptions = rh.executePutRequest(
+            "/_plugins/_security/api/tenancy/config",
+            "{\"sign_in_options\": [\"BASIC\", \"OPENID\"]}",
+            header
+        );
+        assertThat(updateDashboardSignInOptions.getBody(), updateDashboardSignInOptions.getStatusCode(), equalTo(HttpStatus.SC_OK));
+
         getDashboardsinfoResponse = rh.executeGetRequest("/_plugins/_security/dashboardsinfo", ADMIN_FULL_ACCESS_USER);
         assertThat(getDashboardsinfoResponse.getStatusCode(), equalTo(HttpStatus.SC_OK));
         assertThat(getDashboardsinfoResponse.findValueInJson("default_tenant"), equalTo("Private"));
+
+        assertThat(getDashboardsinfoResponse.findArrayInJson("sign_in_options"), hasItem((DashboardSignInOption.BASIC.toString())));
+        assertThat(getDashboardsinfoResponse.findArrayInJson("sign_in_options"), hasItem((DashboardSignInOption.OPENID.toString())));
+
+        final HttpResponse updateUnavailableSignInOption = rh.executePutRequest(
+            "/_plugins/_security/api/tenancy/config",
+            "{\"sign_in_options\": [\"BASIC\", \"SAML\"]}",
+            header
+        );
+        assertThat(updateUnavailableSignInOption.getStatusCode(), equalTo(HttpStatus.SC_BAD_REQUEST));
```

Comment:
this should probly return a 2xx instead of 400 (BAD_REQUEST) since partial update was successful. This return code is mis-leading. This could be covered as a separate PR since this code-change doesn't touch the return code for the associated API.

Reply:
@DarshitChanpura if it doesn't find the option it throws an exception and it doesn't update the sign in options. I created an assertion after the bad request to make sure the sign in options were not updated. 
I chose to return 400, because if that happens, it means there must be some problem in the interface because that option should not be available for selection.
When the admin goes to the Security Dashboard sign-in options, the options available to select are taken from the authc options available in backend [Getting sign-in options in security dashboard](https://github.com/davidosorno/security-dashboards-plugin/blob/33c1c4381148a16dcc75e2caa1d734dcd94fbdde/public/apps/configuration/panels/auth-view/dashboard-signin-options.tsx#L81)

Reply:
I see. so no options will be updated even if 1 succeeded and other failed

Reply:
Returning 400 if any part of the request is invalid seems correct to me- I don't think there should be any changes here.  Good work @davidosorno